tests/unit/amo/components/TestAddAddonToCollection.js

```diff
@@ -38,7 +38,7 @@ describe(__filename, () => {
 
   const getProps = (customProps = {}) => {
     return {
-      addon: createInternalAddon(fakeAddon),
+      addon: createInternalAddonWithLang(fakeAddon),
```

Comment:
Awesome. I was thinking about doing something like that too.

Nit: I'd have used a slightly different name like `createInternalAddonWithDefaultLang` because, to me, `WithLang` means we'd pass a lang to the function.

Reply:
As you saw later, this function does accept a `lang`, it's just that most of the time we don't need to specify one so we don't pass it and it uses a default. Can I mark this comment as resolved?

Reply:
> Can I mark this comment as resolved?

yes